File: atcoder/atcoder_problems/abc/abc001~100/abc051~100/abc061/abc061_b.py
N, M = map(int, input().split())
A_B_roads = [0 for _ in range(N)]

for _ in range(M):
    a, b = map(int, input().split())
    A_B_roads[a - 1] += 1
    A_B_roads[b - 1] += 1

for i in range(N):
    print(A_B_roads[i])

File: atcoder/atcoder_problems/abc/abc001~100/abc051~100/abc061/abc061_b.cpp
#include <iostream>
#include <vector>
using namespace std;

int main (){
    int N, M;
    cin >> N >> M;
    vector<int> A_B_roads(N, 0);

    for (int i = 0; i < M; i++){
        int a, b;
        cin >> a >> b;
        A_B_roads[a - 1] += 1;
        A_B_roads[b - 1] += 1;
    }
    for (int i = 0; i < N; i++) cout << A_B_roads[i] << endl;
    return 0; 
}